vignettes/standard-workflow.Rmd

```diff
@@ -24,7 +24,7 @@ knitr::opts_knit$set(
 library(esqlabsR)
 ```
 
-### Defining scenarios
```

Comment:
@Felixmil just tagging you along, you can read the docs if you want to. The other changes are coming from the other PR, gut mixed up a bit.